Fixed typo :)

pH-7 opened the pull request:
Just noticed a minor typo on your README file.

anderseknert commented:
Thanks @pH-7 ! Are we sure it's a typo though? 🙂 The description in that repo uses "distributor" as well https://github.com/segrax/openpolicyagent

> This library provides a client for the Open Policy Agent (OPA), a PSR-15 authorization middleware and a PSR-15 bundle distributor middleware.

pH-7 commented:
> Thanks @pH-7 ! Are we sure it's a typo though? 🙂 The description in that repo uses "distributor" as well https://github.com/segrax/openpolicyagent
> 
> > This library provides a client for the Open Policy Agent (OPA), a PSR-15 authorization middleware and a PSR-15 bundle distributor middleware.

Ouch, sorry, for some reason, I replaced "distributor" and not "distrubtion". I did the change too quickly. Now fixed on the correct typo 😃

anderseknert commented:
Oh, now that _is_ a typo :) Thanks @pH-7!
